packages/z/zmqpb/xmake.lua

```diff
@@ -0,0 +1,22 @@
+package("zmqpb")
+    set_homepage("https://github.com/SFGrenade/ZmqPb/")
+    set_description("A helper to use zeromq and protobuf together")
+    set_license("MPL-2.0")
+
+    set_urls("https://github.com/SFGrenade/ZmqPb/archive/refs/tags/v$(version).tar.gz",
+             "https://github.com/SFGrenade/ZmqPb.git")
+    add_versions("0.1", "4a34ec92faa381306356e84e2a2000093d8f76cfa037db1f4cd0adb0205faebb")
+
+    on_load(function (package)
+        package:set("installdir", path.join(os.scriptdir(), package:plat(), package:arch(), package:mode()))
```

Comment:
you should use on_install

Reply:
refer https://github.com/xmake-io/xmake-repo/blob/81096aaf98fc85480a1e9aa341aaeee16e363373/packages/d/discord/xmake.lua#L28

Reply:
refer zlib, libspng package

Reply:
added